internal/docstore/drivertest/drivertest.go

```diff
@@ -815,7 +817,9 @@ func addQueryDocuments(t *testing.T, coll *ds.Collection) {
 }
 
 func testGetQuery(t *testing.T, coll *ds.Collection) {
-	cleanUpTable(t, newHighScore, coll)
+	if err := cleanUpTable(newHighScore, coll); err != nil {
```

Comment:
I thought cleanUpTable takes a `t` and calls `t.Fatal` itself?

Reply:
Yes, I wanted to use it with benchmarks as well. Moved to a separate PR.